Makefile

```diff
@@ -1,11 +1,11 @@
 .DEFAULT_GOAL := local
 
 ifndef VERSION
-	VERSION = ${shell git describe --tags --always}
-endif
-
-ifeq ($(VERSION),stable)
-	VERSION = stable-${shell git rev-parse --short HEAD}
+	ifeq (${shell git rev-list -n 1 tags/stable},${shell git rev-parse HEAD}) # if stable tag is the same as current commit
```

Comment:
This will show an error to the user if the stable tag is not existing: 
```
fatal: ambiguous argument 'tags/stable': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

I expect this will happen to most of the users since tags have to be fetch explicitly. Maybe we can suppress the error if no tags are available.